content/kubermatic/master/architecture/known_issues/_index.en.md

```diff
@@ -0,0 +1,64 @@
++++
+title = "Known Issues"
+date = 2022-07-22T12:22:15+02:00
+weight = 25
+
```

Comment:
Now that we already have 2 issues, maybe it would make sense to add a "Table Of Content" on the top of the page. Adding this to the page header should do the trick:
```
enableToc = true
```